### PR TITLE
11 language

### LINKED ISSUE #11: Feature request : add language info and link to repo for users
It could be great to add language name and a link to the user's repository if any.  

It could be done by allowing to bundle a JSON containing id => language / repo url.  
For example :  
```
{
  "userInfo": {
      "12345" : {
         "language" : "Java",
         "repo": "https://github.com/xxxx/xxxx"
    }
  }
}
```
I don't see any way to do it more automatically (e.g letting users declare their own language / repo url) but feel free to suggest one :)

### PR DESCRIPTION
**Pull request by Cpavrai**
This closes #11 with updating README for this new feature.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,22 @@ Key : `mediaSection.location`
 Example : `file:/home/user/customMediaSection.json`  
 Default : `classpath:public/json/media.json`
 
+### Override Users' Profile (optional)
+
+You can override the users' profile informations by setting the `profiles.location` property to your config file.
```
Comment (olevitt) [inline README.md:61]: Should be `user's` instead of `users'`

File: src/main/java/fr/insee/aventcalendar/model/leaderboard/UsersProfile.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package fr.insee.aventcalendar.model.leaderboard;
+
+import java.util.Map;
+
+public class UsersProfile {
```
Comment (olevitt) [inline src/main/java/fr/insee/aventcalendar/model/leaderboard/UsersProfile.java:5]: Not sure if it should be `UsersProfile` or `UserProfiles`

File: README.md [hunk 3]
```diff
@@ -56,6 +56,22 @@ Key : `mediaSection.location`
...
+
+Key : `profiles.location`  
+Example : `file:/home/user/customProfiles.json`  
+Default : `classpath:public/json/profiles.json`
+
+### Override Language Style (optional)
+
+You can override the languages' styles' configuration file's informations by setting the `languageStyle.location` property to your config file.
```
Comment (olevitt) [inline README.md:69]: Check `'` usages here

File: src/main/resources/templates/index.ftl [hunk 4]
```diff
@@ -11,6 +11,7 @@
     <script type="text/javascript" src="bootstrap/js/bootstrap.min.js"></script>
 
     <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <link rel="stylesheet" type="text/css" href="css/language-pills.css" />
```
Comment (olevitt) [inline src/main/resources/templates/index.ftl]: Are you sure you included the corresponding CSS file ?

**Review by olevitt — CHANGES_REQUESTED** [review]
Looks good for the most part, some typos and a possible css missing

**Review by olevitt — CHANGES_REQUESTED** [review]
Profiles data should depend on the year. 
E.G : I did last year using kotlin on bitbucket and I'm doing this year using javascript on github

File: README.md (hunk 1 above)
Comment (Cpavrai) [reply README.md:61]: It is users in plurial, so it is correct

File: src/main/java/fr/insee/aventcalendar/model/leaderboard/UsersProfile.java (hunk 2 above)
Comment (Cpavrai) [reply src/main/java/fr/insee/aventcalendar/model/leaderboard/UsersProfile.java:5]: Every user has his own profile, and there is several users, so...

File: src/main/resources/public/json/profiles.json [hunk 5]
```diff
@@ -2,7 +2,10 @@
   "profiles": {
     "123456789": {
       "link": "https://github.com/123456789",
```
Comment (olevitt) [inline src/main/resources/public/json/profiles.json]: link should also depend on the year. User may use different repositories across years

**Review by olevitt — CHANGES_REQUESTED** [review]

File: src/main/resources/public/json/profiles.json (hunk 5 above)
Comment (Cpavrai) [reply src/main/resources/public/json/profiles.json]: I don't think so. It is just a link to their github's/other interface's profile

Comment (olevitt) [reply src/main/resources/public/json/profiles.json]: Nah, users tend to link to their repo and make a new repo each year

Comment (Cpavrai) [reply src/main/resources/public/json/profiles.json]: Ok it is resolved from now

**Review by olevitt — APPROVED** [review]
Yes, looks good to me :+1:

**Pull request merged**